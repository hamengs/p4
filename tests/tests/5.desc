vmalloc multiple small page allocation
