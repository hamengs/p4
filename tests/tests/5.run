cd ../solution; ../tests/run-xv6-command.exp CPUS=4 Makefile.test test_5 | grep -E 'XV6_TEST_OUTPUT|XV6_TEST_ERROR'; rm *test_5* ; cd ../tests
