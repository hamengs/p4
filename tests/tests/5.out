XV6_TEST_OUTPUT Start of program - base pages:3 huge pages:0
XV6_TEST_OUTPUT After first vmalloc, 100 bytes - program base pages:11 huge pages:0
XV6_TEST_OUTPUT After second vmalloc, 5000 bytes - program base pages:11 huge pages:0
